CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Release Notes
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+## Platform Version 0.11.11 - UNRELEASED
```

Comment:
Release with 0.11.10 at the top, add the unreleased line later.  Other than that LGTM

Reply:
fixed!